sections/semantics-document-metadata.include

```diff
@@ -355,15 +355,16 @@
     <dd>
       <pre class="idl" data-highlight="webidl">
         interface HTMLLinkElement : HTMLElement {
-          attribute DOMString href;
-          attribute DOMString? crossOrigin;
-          attribute DOMString rel;
-          attribute DOMString rev;
-          [SameObject, PutForwards=value]readonly attribute DOMTokenList relList;
-          attribute DOMString media;
-          attribute DOMString hreflang;
-          attribute DOMString type;
-          [SameObject, PutForwards=value] readonly attribute DOMTokenList sizes;
+          [CEReactions] attribute USVString href;
```

Comment:
If we're going to use CEReactions, it would probably be a good idea to make sure that they are defined somewhere.